charts/traffic/values.yaml

```diff
@@ -55,4 +55,8 @@ resources:
 storage: 50Mi
 storageClassName: standard
 
-persist: false
\ No newline at end of file
+persist: false
```

Comment:
If `persist` is gone, shouldn't it also be removed here?

Reply:
done